src/test/kotlin/org/rust/ide/annotator/RsUnsafeExpressionErrorAnnotatorTest.kt

```diff
@@ -132,4 +134,33 @@ class RsUnsafeExpressionErrorAnnotatorTest : RsAnnotatorTestBase(RsUnsafeExpress
             foo();
         }
     """)
+
+    fun `test need unsafe asm macro call`() = checkErrors("""
+       fn main() {
+            <error descr="use of `asm!` is unsafe and requires unsafe function or block [E0133]">asm!("nop");</error>
+       }
+    """)
+
+    fun `test need unsafe asm macro expr`() = checkErrors("""
+       fn main() {
+            <error descr="use of `asm!` is unsafe and requires unsafe function or block [E0133]">asm!("nop")</error>
+       }
+    """)
+
+    @ProjectDescriptor(WithDependencyRustProjectDescriptor::class)
+    fun `test a repeatedly defined asm macro`() = checkErrors("""
```

Comment:
@Undin Can you see this test case? My local test failed and CI failed. Did I make a mistake?